### PR TITLE
Make a link clickable on alternate downloads page

### PR DESCRIPTION
**Pull request by pbo-linaro**

### CODE REVIEW

**Comment by directionless** [comment]
I tried that -- the whole value ends up url escaped. I can give it another shot later today. You can also run the site locally in dev mode. It's a little janky, but there are notes in the README

**Comment by pbo-linaro** [comment]
React.js sanitizes displayed strings and escape all characters to prevent issues with user inputs (XSS attack, ...).
Since only static content is printed in `Paragraph` sections (from our json files), we can safely display raw html instead.

First commit enables this, second one adds the clickable link.

**Comment by pbo-linaro** [comment]
Renders as:
out

File: src/components/text/Paragraph/Paragraph.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Paragraph = ({ children, className, highlight }) => {
     [`${baseClass}--highlight`]: highlight,
   })
 
-  return <p className={classes}>{children}</p>
+  return <p className={classes} dangerouslySetInnerHTML={{ __html: children }}></p>
```
Comment (directionless) [inline src/components/text/Paragraph/Paragraph.js:14]: I think this is right, but I'm worried about applying it everywhere. What if we created a new class, `DangerousParagraph` and applied it to this page?

Comment (pbo-linaro) [reply src/components/text/Paragraph/Paragraph.js:14]: Only home and download pages are using it, so I would not be too afraid of this.
If you prefer a whole new component, you're welcome to add it.

**Pull request closed without merging**